src/types/docker.ts

```diff
@@ -8,6 +8,8 @@ export class DockerExec {
    }
 
    public async pull(image: string, args: string[], silent?: boolean) {
+      image = image.replace(/^[ ]+/g, '') //Remove whitespace at beginning of the string
```

Comment:
whitespace at the beginning and end of the string doesn't matter. these two additions are unnecessary